nginx_error_page/error.html

```diff
@@ -0,0 +1,415 @@
+        <path stroke="#42263a" d="M14 32h2m-2 1h2" />
+        <path stroke="#b0649a" d="M16 32h2m-2 1h2" />
+        <path stroke="#975685" d="M22 32h2m-2 1h2" />
+        <path stroke="#1c1018" d="M24 32h2m-2 1h2" />
+        <path stroke="rgba(0,0,0,0.8862745098039215)" d="M26 33h1" />
+        <path stroke="rgba(0,0,0,0.6039215686274509)" d="M27 33h1" />
+        <path stroke="rgba(0,0,0,0.10980392156862745)" d="M8 34h2m-2 1h1" />
+        <path stroke="#804970" d="M14 34h2m-2 1h2" />
+        <path stroke="#844b73" d="M16 34h2m-2 1h2" />
+        <path stroke="#6e3e60" d="M18 34h2m-2 1h2" />
+        <path stroke="rgba(0,0,0,0.8274509803921568)" d="M25 34h1" />
+        <path stroke="rgba(0,0,0,0.023529411764705882)" d="M26 34h1" />
+        <path stroke="rgba(0,0,0,0.5019607843137255)" d="M9 35h1m12 1h1" />
+        <path stroke="rgba(0,0,0,0.7098039215686275)" d="M24 35h1" />
+        <path stroke="rgba(0,0,0,0.5137254901960784)" d="M25 35h1m-5 2h1" />
+        <path stroke="rgba(0,0,0,0.9333333333333333)" d="M21 36h1" />
+        <path stroke="rgba(0,0,0,0.16470588235294117)" d="M23 36h1" />
+        <path stroke="rgba(0,0,0,0.23137254901960785)" d="M8 37h2" />
+        <path stroke="rgba(0,0,0,0.807843137254902)" d="M12 37h1" />
+        <path stroke="rgba(0,0,0,0.8549019607843137)" d="M13 37h1" />
+        <path stroke="rgba(0,0,0,0.9372549019607843)" d="M14 37h2" />
+        <path stroke="rgba(0,0,0,0.9411764705882353)" d="M16 37h2" />
+        <path stroke="rgba(0,0,0,0.8745098039215686)" d="M18 37h1" />
+      </svg>
+
+      <h1>
+        <svg
+          width="50"
+          height="30"
+          viewBox="0 0 38 38"
+          xmlns="http://www.w3.org/2000/svg"
+          class="spinner"
+          stroke="#000"
+        >
+          <g fill="none" fill-rule="evenodd">
+            <g transform="translate(1 1)" stroke-width="2">
+              <circle stroke-opacity=".5" cx="18" cy="18" r="18" />
+              <path d="M36 18c0-9.94-8.06-18-18-18">
+                <animateTransform
+                  attributeName="transform"
+                  type="rotate"
+                  from="0 18 18"
+                  to="360 18 18"
+                  dur="1s"
+                  repeatCount="indefinite"
+                />
+              </path>
+            </g>
+          </g>
+        </svg>
+        Starting Kolibri server...
+      </h1>
+
+      <p class="message">
+        If you are not redirected in a few minutes, please contact our community forums or try refreshing this page.
+      </p>
+      <div>
+        <a href="https://community.learningequality.org/c/support/kolibri">Ask for help on our community forums</a>
+      </div>
+      <button class="raised" onClick="location.reload();">REFRESH</button>
+      <div>kolibri-server 1.2.3</div>
+    </div>
+
+    <script>
+      function retry() {
+        let req = new XMLHttpRequest()  // use it instead of fetch for IE11 compatibility
```

Comment:
`var`?

Reply:
yes, that's what i meant

Reply:
`let` is available in IE11 that's our bottom browser recommendation. I don't see any problem in using `var`here as the code is short, but there should not be any problem using let either.

Reply:
My general philosophy is that _if it's easy_ to support older browsers than we claim to support, then we should make a best effort to do it. It seems like the "right" thing to do in case people are trying to get things running and haven't yet upgraded their browsers.

I expressed a similar sentiment to @bjester the other day in this PR: https://github.com/learningequality/kolibri/pull/5631#pullrequestreview-304858645

I'm getting the sense that not everyone agrees, so perhaps this is something we should come to consensus on the team about before I leave more review comments.

Reply:
Yes, it makes sense. I will change it.

Reply:
@indirectlylit all the requested comments and changed have been done. Apart from these minor details there was an important comment by @bjester that I apply to improve the `nginx.conf` file. So this PR is ready to be re-reviewed again.